docker/test/style/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN apt-get update && env DEBIAN_FRONTEND=noninteractive apt-get install --yes \
     shellcheck \
     yamllint \
     locales \
-    && pip3 install black==23.1.0 boto3 codespell==2.2.1 mypy==1.3.0 PyGithub unidiff pylint==2.6.2 \
```

Comment:
You won't believe.. but we need bring it back, and leave "FIXME" comment here to get off rid this package in a couple of months. As well as in Ami script.

Old PRs will break horribly.

Besides that, we need to backport the changes everywhere.